packages/components/src/OverflowTypography/OverflowTypography.stories.tsx

```diff
@@ -88,3 +88,20 @@ export const TooltipProps = () => {
     </div>
   );
 };
+
+/**
+ * Prop `visibleLastSymbolsCount` позволяет задавать число отображаемых символов после сокращения,
+ * при использовании `visibleLastSymbolsCount` не работает props `rowsCount`
```

Comment:
```suggestion
 * При использовании `visibleLastSymbolsCount` не работает props `rowsCount`
```

Пропс не работает или работает не корректно? Если второе, то так и напиши, что поддеживается работа только для отображения в одну строку

Reply:
Пропс сейчас не работает